src/main.rs

```diff
@@ -1,9 +1,39 @@
 use std::env;
+use settings::*;
+use std::str::FromStr;
+use std::process::exit;
+
+#[macro_use]
+extern crate lazy_static;
+#[macro_use]
+extern crate clap;
 
 mod branch;
 mod changes;
+mod settings;
+
 
 fn main() -> std::io::Result<()> {
+    let mut app = clap_app!(rustygitprompt =>
+        (version: "1.0")
+        (author: "Mikołaj Baranowski <mikolajb@gmail.com>")
+        (about: "A simple git prompt in rust")
+        (@arg FORMAT: -i --icon_list_format +takes_value "Sets the format for icon list: superscript, subscript or digits (default)")
+    );
+
+    let format = app.clone().get_matches().value_of("FORMAT").map(|s| s.to_string());
```

Comment:
borrow checker is annoying here, so have to convert it to String so the Copy trait triggers